### ISSUE TITLE
Allow multiple files in the --ignore-path flag

### ISSUE THREAD

**Issue by johnbillion** [stylelint/stylelint#4680]
> What is the problem you're trying to solve?

My project contains a `.gitignore` file that I'd like to use as the basis for my stylelint ignore rules. I can do this with `--ignore-path .gitignore`,  however I'd like to supplement this with additional ignore rules inside `.stylelintignore`.

> What solution would you like to see?

I would like to be able to specify more than one ignore file and have the rules combined. For now I think that I can get around this by using a combination of the `--ignore-path .gitignore` flag plus the `ignoreFiles` config option, but I've been lead to believe that the latter is not an efficient way to ignore files.

Would it be possible to allow more than one file in the `--ignore-path` flag? (Or multiple uses of the `--ignore-path` flag to create an array)

**Comment by jeddy3** [comment]
@johnbillion Thanks for the request and for using the template.

Funnily enough, I ran into this myself when configuring Prettier in one of the stylelint org repos. I wanted to add ignore rules on top of the ones in `.gitignore`.

With ESLint and Prettier (since v2) supporting `.`, and with [our intention to do the same](https://github.com/stylelint/stylelint/issues/3411), I can see this becoming a popular request.

> Would it be possible to allow more than one file in the `--ignore-path` flag? 

I think that works. The content of each ignore file would be appended to the one the proceeds it as the order is important because of negation in globs. 

@evilebottnawi (who is a contributor here and on Prettier) [discussed the feature within remark](https://github.com/remarkjs/remark/issues/416) and some concerns were raised, so we'll need to make sure we account for those if we go ahead and implement this feature.

I'll label as "status: needs discussion" to see if any else has any thoughts.

**Comment by jeddy3** [comment]
As there are no objections, I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

> (Or multiple uses of the `--ignore-path` flag to create an array)

Let's go with multiple uses of the flag as that seems to be a [common](https://prettier.io/docs/en/plugins.html#using-plugins) [pattern](https://eslint.org/docs/user-guide/command-line-interface#specifying-rules-and-plugins), rather than have `ignore-path` accept multiple paths.

**Comment by AndreasNasman** [comment]
This is hasn't been closed by another issue, i.e. implemented already, correct?

I'm developing in a mono-repo containing multiple projects, and this feature could help simplify and unify code ignores greatly. Basing ignores on both `.gitignore` and `.stylelintignore` files is specifically what I'm looking for.

**Comment by jeddy3** [comment]
> This is hasn't been closed by another issue, i.e. implemented already, correct?

Correct.

**Comment by kimulaco** [comment]
I would like to work on this issue.
I will work to support the following command.

```bash
stylelint styles/*.css --ignore-path .stylelintignore_a --ignore-path .stylelintignore_b
```
